infra/discovery/src/listener/webhooks.js

```diff
@@ -40,11 +40,11 @@ async function postToDiscordWebhook(url, data) {
   }
 
   const icon = eventIcons[data.log.eventName] || ':dromedary_camel: '
-  const listing = data.related.listing
+  const listing = data.listing
```

Comment:
The GCF for IPFS pinner relies on the related field so I think we'll need to update it.